app/src/marketplace/modules/product/services.js

```diff
@@ -13,12 +13,15 @@ import { getValidId, mapProductFromApi } from '$mp/utils/product'
 import { getProductFromContract } from '$mp/modules/contractProduct/services'
 import getWeb3 from '$shared/web3/web3Provider'
 
-export const getProductById = async (id: ProductId): ApiResult<Product> => get(formatApiUrl('products', getValidId(id, false)))
+export const getProductById = async (id: ProductId): ApiResult<Product> => get({
+    url: formatApiUrl('products', getValidId(id, false)),
```

Comment:
I think this should also be `useAuthorization: false`. Getting a product by ID does not require Auth.

Reply:
First I figured that as well but that led to an issue where draft products cannot be fetched, ie. create a product -> cannot access that product in the editor.

Reply:
Ah, good point. Wouldn't this cause us problems with `/products/${id}/streams` too - while I'm editing a draft product? I can test that on Monday morning but at least on production it 403s for draft products.

<img width="462" alt="Screenshot 2019-11-30 at 9 28 51" src="https://user-images.githubusercontent.com/1593398/69898052-5ad04280-1354-11ea-931c-d71062851a3a.png">

Reply:
@fonty1 damn just merged this :) I think you are right... I'll do a fix for it in a separate PR. At least for the streams we can figure out if we need auth or not since the product will be known. For the product we'd need to fetch it first before knowing if it's draft or not.

Reply:
👍 yep seems like the best way to do it!